apps/site/components/Common/BlogPostCard/index.stories.tsx

```diff
@@ -11,12 +11,7 @@ export const Default: Story = {
     category: 'vulnerability',
     description:
       'Starting on March 15th and going through to March 17th (with much of the issue being mitigated on the 16th), users were receiving intermittent 404 responses when trying to download Node.js from nodejs.org, or even accessing parts of the website.',
-    authors: [
-      {
-        fullName: 'Hayden Bleasel',
-        src: 'https://avatars.githubusercontent.com/u/',
-      },
-    ],
+    authors: ['Hayden Bleasel'],
```

Comment:
```suggestion
    authors: ['canerakdas'],
```

story with valid GitHub username

Reply:
This component requires an author name, I updated it with one that is in the authors file